### PR TITLE
Merge 4.8.1 into 4.9.0

### PR DESCRIPTION
**Pull request by fcaffieri**
| Related issue |
|:--:|
| https://github.com/wazuh/wazuh-qa/issues/5549 |

## Description
Merge 4.8.1 into 4.9.0

### CODE REVIEW

File: .github/workflows/code_analysis.yaml
Comment (rauldpm) [inline .github/workflows/code_analysis.yaml:1]: File removed in https://github.com/wazuh/wazuh-qa/pull/4879 (4.9.0), should not be added

File: .github/workflows/scripts/linter.py
Comment (rauldpm) [inline .github/workflows/scripts/linter.py:1]: File removed in https://github.com/wazuh/wazuh-qa/pull/4879 (4.9.0), should not be added

File: provisioning/roles/wazuh/ansible-filebeat-oss/defaults/main.yml [hunk 1]
```diff
@@ -1,7 +1,11 @@
 ---
 filebeat_version: 7.10.2
 
+<<<<<<< HEAD
+wazuh_template_branch: "{{ SOURCE_REFERENCE }}"
+=======
 wazuh_template_branch: 4.9.0
+>>>>>>> 5a460cbb266f502fd2376326ab69cf7ab35759da
```
Comment (rauldpm) [inline provisioning/roles/wazuh/ansible-filebeat-oss/defaults/main.yml:9]: Bad conflict resolution

**Review by rauldpm — CHANGES_REQUESTED** [review]

File: provisioning/roles/wazuh/ansible-filebeat-oss/defaults/main.yml (hunk 1 above)
Comment (fcaffieri) [reply provisioning/roles/wazuh/ansible-filebeat-oss/defaults/main.yml]: Done in [3624d76](https://github.com/wazuh/wazuh-qa/pull/5552/commits/3624d7627e9dc6d1fd82ace2ed034ded3b32fb32)

File: .github/workflows/scripts/linter.py
Comment (fcaffieri) [reply .github/workflows/scripts/linter.py:1]: Done in [3624d76](https://github.com/wazuh/wazuh-qa/pull/5552/commits/3624d7627e9dc6d1fd82ace2ed034ded3b32fb32)

File: .github/workflows/code_analysis.yaml
Comment (fcaffieri) [reply .github/workflows/code_analysis.yaml:1]: Done in [3624d76](https://github.com/wazuh/wazuh-qa/pull/5552/commits/3624d7627e9dc6d1fd82ace2ed034ded3b32fb32)

**Review by rauldpm — APPROVED** [review]

**Pull request merged**